### ISSUE TITLE
Pattern(?): Using aria-labelledby with Lists

### ISSUE THREAD

**Issue by svinkle** [a11yproject/a11yproject.com#350]
Not sure which category this should be placed under, or if it's a pattern at all? It's an example of using `aria-labelledby` with lists to help give more context. http://codepen.io/svinkle/pen/eNPyEB

@davatron5000 @dennisgaebel @mpiotrowicz @joe-watkins

**Comment by dennisgaebel** [comment]
I'm gonna go with Pattern.

**Comment by davatron5000** [comment]
It's a pretty good example of labelling. Maybe we make a new "Lists" section. Maybe that's too vague.

**Comment by svinkle** [comment]
I think you hit the nail on the head, "Labelling". A category of pens showcasing how and when to use things like `aria-label`, `aria-labelledby` and/or `aria-describedby`. I often get these confused myself.

Thoughts?

**Comment by davatron5000** [comment]
:+1:
